runtime/src/registry/types.rs

```diff
@@ -0,0 +1,151 @@
+//pub use crate::nft::AssetId;
+use frame_support::dispatch;
+use codec::{Decode, Encode};
+use sp_std::{vec::Vec, fmt::Debug};
+use sp_core::{U256, H160};
+use crate::proofs;
+
+pub const NFTS_PREFIX: &'static [u8] = &[20, 0, 0, 0, 0, 0, 0, 1];
+
+/// A vector of bytes, conveniently named like it is in Solidity.
+pub type Bytes = Vec<u8>;
+
+/// Registries are identified using a nonce in storage.
+pub type RegistryId = H160;
+
+/// A cryptographic salt to be combined with a value before hashing.
+pub type Salt = Bytes;
```

Comment:
Salt is Byte32 array, not sure if you want to constrain it here.

Reply:
Updated 👍